library/src/scala/tasty/reflect/SymbolOps.scala

```diff
@@ -143,6 +143,16 @@ trait SymbolOps extends Core { selfSymbolOps: FlagsOps =>
     /** The symbol of the companion module */
     def companionModule(given ctx: Context): Symbol =
       internal.Symbol_companionModule(self)
+
+    def typeRef(given ctx: Context): TypeOrBounds =
+      internal.Symbol_typeRef(self)
+
+    def termRef(given ctx: Context): TypeOrBounds =
+      internal.Symbol_termRef(self)
+
+    def info(given ctx: Context): TypeOrBounds =
+      internal.Symbol_info(self)
+
```

Comment:
Maybe add documentation that programmers should prefer `Type.memberType` for memeber types, and use the above for local definitions.

Reply:
Most of the original names we invented so far have reverted back to the name in the compiler. It is much simpler to track what is what.

Reply:
I didn't make it clear, I don't suggest to rename. We have a method in tasty reflect `memberType`, which can be used to get the type of a member; whose functionality overlaps with the methods here. If I understand correctly, the methods here are mainly intended for local definitions instead of member definitions.

Reply:
That is unrelated to this PR. Let's look at it afterwards.